.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+install:
+  - "pip install -r requirements.txt"
+  - "pip install -r test-requirements.txt"
+  - "pip install flake8"
+before_script:
+ - flake8 .
+script:
+ - tests/test-flake8.sh
```

Comment:
This should be in before_script. That way, when we add functionality tests they can go in the script section.